paparazzi/gradle.properties

```diff
@@ -16,6 +16,7 @@ POM_DEVELOPER_URL=https://github.com/cashapp/
 
 SONATYPE_HOST=DEFAULT
 RELEASE_SIGNING_ENABLED=true
+SONATYPE_AUTOMATIC_RELEASE=true
```

Comment:
is this only on snapshot builds or release builds too?  

i eventually want this, when a new git tag is pushed (similar to how some other OSS repos do it) but not sure if this repo is ready for this yet?  @JakeWharton thoughts?

Reply:
This is for release builds, snapshots are automatically available after publishing either way. I can set it to false for now if you want to but from what I can see everything you should need for it is in place (I've also used this to publish a fork from master for us a few times, although that was using a different github actions workflow).

Reply:
let's see how it goes!